Fails to build with cocogitto

**Describe the bug**

Fails to build with latest cocogitto:

```
   Compiling cocogitto v6.1.0
   Compiling koji v2.2.0
error[E0061]: this method takes 1 argument but 7 arguments were supplied
  --> /Users/berkus/.cargo/registry/src/index.crates.io-6f17d22bba15001f/koji-2.2.0/src/lib/commit.rs:44:15
   |
44 |     cocogitto.conventional_commit(
   |               ^^^^^^^^^^^^^^^^^^^
45 |         &commit_type,
   |         ------------ expected `CommitOptions<'_>`, found `&String`
   |
note: method defined here
  --> /Users/berkus/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cocogitto-6.1.0/src/command/commit.rs:24:12
   |
24 |     pub fn conventional_commit(&self, opts: CommitOptions) -> Result<()> {
   |            ^^^^^^^^^^^^^^^^^^^
help: remove the extra arguments
   |
45 -         &commit_type,
45 +         /* CommitOptions<'_> */,
   |

For more information about this error, try `rustc --explain E0061`.
error: could not compile `koji` (lib) due to 1 previous error
```

**To Reproduce**

```
cargo +nightly install koji
```

**Expected behavior**

Koji should install.

**Additional context**

Not sure when cocogitto changed this interface, but it seems like they broke API between 6.0 and 6.1. Koji should probably depend strictly on 6.0 or upgrade the dep to 6.1.

--- a/.github/workflows/call-build-upload.yml
+++ b/.github/workflows/call-build-upload.yml
@@ -41,6 +41,7 @@ jobs:
     name: ${{ matrix.target }}
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         include:
           - target: aarch64-unknown-linux-gnu

--- a/.github/workflows/call-build-upload.yml
+++ b/.github/workflows/call-build-upload.yml
@@ -12,6 +12,16 @@ on:
       tag_ref:
         required: true
         type: string
+  workflow_dispatch:
+    inputs:
+      package_name:
+        description: The name of the package (e.g. 'koji')
+        required: true
+        type: string
+      tag_ref:
+        description: The tag reference (e.g. 'refs/tags/v0.1.0')
+        required: true
+        type: string
 
 env:
   CARGO_INCREMENTAL: 0
@@ -64,4 +74,5 @@ jobs:
           target: ${{ matrix.target }}
           tar: all
           zip: windows
+          features: ${{ contains(matrix.target, 'windows') && 'vendored-openssl' || '' }}
           token: ${{ secrets.GITHUB_TOKEN }}

--- a/.github/workflows/push-release-plz.yml
+++ b/.github/workflows/push-release-plz.yml
@@ -38,5 +38,6 @@ jobs:
       package_name: ${{ matrix.release.package_name }}
       tag_ref: refs/tags/${{ matrix.release.tag }}
     strategy:
+      fail-fast: false
       matrix:
         release: ${{ fromJson(needs.release-plz.outputs.releases) }}

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -2,7 +2,10 @@
 name = "koji"
 version = "2.2.0"
 edition = "2021"
-authors = ["Finley Thomalla <finley@thomalla.ch>", "Danny Tatom <its.danny@hey.com>"]
+authors = [
+  "Finley Thomalla <finley@thomalla.ch>",
+  "Danny Tatom <its.danny@hey.com>",
+]
 description = "An interactive CLI for creating conventional commits."
 documentation = "https://docs.rs/koji"
 repository = "https://github.com/cococonscious/koji"
@@ -19,15 +22,18 @@ path = "src/lib/lib.rs"
 [dependencies]
 anyhow = "1.0"
 clap = { version = "4.4", features = ["derive"] }
-cocogitto = { version = "6.0", default-features = false }
+cocogitto = { version = "< 6.1", default-features = false }
 conventional_commit_parser = "0.9"
 dirs = "5.0"
 emojis = "0.6"
-git2 = { version = "0.18", features = ["vendored-openssl"] }
+git2 = { version = "0.18" }
 indexmap = "2.1"
 requestty = "0.5"
 serde = { version = "1.0", features = ["derive"] }
 toml = "0.8"
+
+[features]
+vendored-openssl = ["git2/vendored-openssl"]
 
 [dev-dependencies]
 rusty-hook = "0.11"

